UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.IntegrationTests/RadioNavigationalWarning/Admin/RadioNavigationalWarningAdminTest.cs

```diff
@@ -0,0 +1,158 @@
+            await SeedWarningType(GetFakeWarningTypes());
+        }
+
+        [SetUp]
+        public void Setup()
+        {
+            _fakeLogger = A.Fake<ILogger<RadioNavigationalWarningsAdminController>>();
+            _rnwRepository = new RNWRepository(_fakeContext);
+            _rnwService = new RNWService(_rnwRepository, _fakeRadioNavigationalWarningConfiguration, _fakeLoggerRnwService);
+
+            _controller = new RadioNavigationalWarningsAdminController(_fakeHttpContextAccessor, _fakeLogger, _rnwService);
+        }
+
+        [Test]
+        public async Task WhenCallIndex_ThenReturnListAsync()
+        {
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+            IActionResult result = await _controller.Index(1, null, null);
+            RadioNavigationalWarningsAdminFilter adminListFilter = (RadioNavigationalWarningsAdminFilter)((ViewResult)result).Model;
+
+            Assert.IsTrue(adminListFilter.RadioNavigationalWarningsAdminList.Count == 8);
+            Assert.AreEqual(1, adminListFilter.PageCount);
+            Assert.AreEqual(0, adminListFilter.SrNo);
+            Assert.AreEqual(1, adminListFilter.CurrentPageIndex);
+        }
+
+        [Test]
+        public async Task WhenCallIndexWithWarningTypeFilter_ThenReturnFilteredListAsync()
+        {
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+            IActionResult result = await _controller.Index(1, 1, null);
+            RadioNavigationalWarningsAdminFilter adminListFilter = (RadioNavigationalWarningsAdminFilter)((ViewResult)result).Model;
+            Assert.AreEqual(4, adminListFilter.RadioNavigationalWarningsAdminList.Count);
+            Assert.AreEqual(1, adminListFilter.PageCount);
+            Assert.AreEqual(0, adminListFilter.SrNo);
+            Assert.AreEqual(1, adminListFilter.CurrentPageIndex);
+        }
+
+        [Test]
+        public async Task WhenCallIndexWithYearFilter_ThenReturnFilteredListAsync()
+        {
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+            IActionResult result = await _controller.Index(1, null, 2020);
+            RadioNavigationalWarningsAdminFilter adminListFilter = (RadioNavigationalWarningsAdminFilter)((ViewResult)result).Model;
+            Assert.AreEqual(2, adminListFilter.RadioNavigationalWarningsAdminList.Count);
+            Assert.AreEqual(1, adminListFilter.PageCount);
+            Assert.AreEqual(0, adminListFilter.SrNo);
+            Assert.AreEqual(1, adminListFilter.CurrentPageIndex);
+        }
+
+        [Test]
+        public async Task WhenCallIndexWithWarningTypeAndYearFilter_ThenReturnFilteredListAsync()
+        {
+            _fakeRadioNavigationalWarningConfiguration.Value.AdminListRecordPerPage = 20;
+            IActionResult result = await _controller.Index(1, 2, 2024);
```

Comment:
The use of '1' for the warning type is obscure. Could you use the constant from the WarningType class?

Reply:
Resolved this review comments in **Feature/46540-integration-testing-setup** branch, Once that branch is merged then I will fix this.